src/main/java/gov/usgs/traveltimeservice/Application.java

```diff
@@ -0,0 +1,18 @@
+package gov.usgs.traveltimeservice;
+
+import io.micronaut.runtime.Micronaut;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Info;
+
+@OpenAPIDefinition(
+    info =
+        @Info(
+            title = "traveltime service",
+            version = "0.0",
+            description = "traveltime service description"))
```

Comment:
An actual description would be more helpful: This is a service to request predicted phase travel times."